### DISCUSSION TITLE
Missing instructions in README?

### DISCUSSION THREAD

**Discussion by guoliang** [lukas-reineke/indent-blankline.nvim#684] [General]
https://github.com/lukas-reineke/indent-blankline.nvim#scope
the setup part is empty, and it looks the same for the other alternatives as well.

**Comment by lukas-reineke** [comment]
That's because you don't need any special configuration for those examples.

**Reply by guoliang** [reply]
aha, makes sense. in my case the current scope doesn't get another color though.
is there a setting property I can use to set the indent guide color for the current selected scope only?

it looks as following for me now
![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/542598/2c1cfcfe-ded8-42f5-8cec-2616b0a5ba98)

**Reply by lukas-reineke** [reply]
You can set the scope color with `:help ibl.config.scope.highlight`

**Reply by guoliang** [reply]
just one last question. is there a setting so that indent line is just showing for current scope, and all others are invisible?

**Reply by lukas-reineke** [reply]
You can set `:help ibl.config.indent.char` to an empty string, and `:help ibl.config.scope.char` to the indent character. Then you'll only see indent for the scope.
